src/modules/pythonmonkey/pythonmonkey.cc

```diff
@@ -174,20 +174,17 @@ static bool getEvalOption(PyObject *evalOptions, const char *optionName, bool *b
   PyObject *value;
 
   value = PyDict_GetItemString(evalOptions, optionName);
-  if (value) {
-    if (PyLong_Check(value))
-      *b_p = PyBool_FromLong(PyLong_AsLong(value));
-    else
-      *b_p = value != Py_False;
-  }
+  if (value)
+    *b_p = PyObject_IsTrue(value) ? true : false;
```

Comment:
```suggestion
    *b_p = PyObject_IsTrue(value) == 1 ? true : false;
```
https://docs.python.org/3/c-api/object.html#c.PyObject_IsTrue
`PyObject_IsTrue` returns `-1` on failure

Reply:
"PyObject_IsTrue returns -1 on failure"

We need a facepalm emoji!  Thanks for catching this.